testgrid/web/src/components/views/InstanceTable.jsx

```diff
@@ -216,7 +226,15 @@ export default class InstanceTable extends React.Component {
     const rows = Object.keys(this.props.instancesMap).map(kurlURL => {
       return (
         <tr key={kurlURL}>
-          <td><span className="url" onClick={() => this.viewInstanceInstaller(this.props.instancesMap[kurlURL][0])}>{kurlURL}</span></td>
+          <td>
+            <span className="url" onClick={() => this.viewInstanceInstaller(this.props.instancesMap[kurlURL][0])}>{kurlURL}</span>
+            {(this.props.instancesMap[kurlURL][0].upgradeURL !== "") &&
```

Comment:
i think this should be the following. @salahalsaleh can you help out here?

```
(this.props.instancesMap[kurlURL][0].upgradeURL) &&
```

Reply:
I can do that, and yeah makes sense

Reply:
yes you're right @emosbaugh, cuz that will also check for `null` & `undefined`.

Reply:
also, no need for the parentheses